examples/src/java/com/twitter/heron/examples/ExclamationTopology.java

```diff
@@ -68,17 +65,9 @@ public static void main(String[] args) throws Exception {
         ExampleResources.getContainerRam(spouts + bolts, parallelism));
     com.twitter.heron.api.Config.setContainerCpuRequested(conf, 1);
 
-    if (args != null && args.length > 0) {
-      conf.setNumWorkers(parallelism);
-      StormSubmitter.submitTopology(args[0], conf, builder.createTopology());
-    } else {
-      System.out.println("Topology name not provided as an argument, running in simulator mode.");
```

Comment:
do we keep the simulator in the heron example?
same comment to the other LocalCluster example

Reply:
Do we really need a simulator in Heron?  A LocalCluster or simulator is useful in Storm because running a single topology requires you start up a cluster on a single node (Nimbus, zookeeper, Supervisor, etc.) which can be a lot of work.  This is not the case in Heron.  It is super easy to run a topology in local mode.  Thus the usefulness for a simulator is diminished.

Reply:
as far as i know, the heron-simulator runs all heron-components in single process for topology writers to easily debug, which is different from multi-process local mode.

Reply:
gotcha, but is it easier to debug if everything just threads in a single process? A jstack might be confusing as there are many threads executing at the same time.  People often say it was hard to debug storm workers because there are many instances/threads of different components running in each

Reply:
but perhaps this discussion about the usefulness of the simulator is not in the scope of this pull request.  I can just add back the code for the simulator and we can discuss the usefulness of having a simulator in heron somewhere else